nav2_msgs/action/FollowPath.action

```diff
@@ -6,3 +6,5 @@ string controller_id
 std_msgs/Empty result
 ---
 #feedback
+float32 distance_to_goal
+float32 speed
```

Comment:
I feel like the current distance from the path could be useful feedback, especially in implementations that use a more control theory oriented implementation like MPC.

Reply:
Perhaps, but that is non-trivial to compute and I don't want to slow down the controller process too much. It'll involve at least a few vector multiplications and a cubic spline.